docs/examples/patterns/navigation/default-dark.html

```diff
@@ -0,0 +1,35 @@
+---
+layout: examples
+title: Navigation / Default - dark
+category: _patterns
+---
+<header id="navigation" class="p-navigation is-dark">
```

Comment:
Why didn't you make dark a modifier?

Reply:
Just because the p-strip is called is-dark. WIl lchange it to --dark.

Reply:
agreed to leave like this